packages/components/organisms/f-checkout/src/components/Accordion.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+    <div>
+        <button
+            :id="`${id}-accordion-header`"
+            :aria-controls="`${id}-header`"
+            :aria-expanded="isExpanded"
+            :class="$style['c-checkout-accordion-header']"
+            :data-test-id="`${id}-accordion-header`"
+            type="button"
+            @click="toggle">
+            {{ title }}
+            <chevron-icon
+                :class="[
+                    $style['c-checkout-accordion-header-arrow'], {
+                        [$style['c-checkout-accordion-header-arrow--expanded']]: isExpanded
+                    }]" />
+        </button>
+        <section
+            :id="`${id}-accordion-section`"
+            :aria-labelledby="`${id}-section`"
```

Comment:
I might be missing it, but where's this section that will be used to label this element?